### PR TITLE
Add details about previewing organisation info

### PR DESCRIPTION
**Pull request by fofr**
The way of previewing org info is quite contrived. Before we make a better org preview feature, indicate to users how they can see their org info before publishing.

TODO:
I need someone to wrap the preview text in the correct feature flag logic so that we only show the "how to preview" text to users who can enrich courses and preview them.

### CODE REVIEW

**Comment by dankmitchell** [comment]
This whole section is already hidden behind a feature flag. See screenshot

**Comment by dankmitchell** [comment]
screen shot 2018-09-03 at 10 57 31

**Comment by the-daniel-rothig** [comment]
@fofr Added feature flag as described

File: src/ui/Controllers/OrganisationController.cs [hunk 1]
```diff
@@ -19,10 +19,12 @@ namespace GovUk.Education.ManageCourses.Ui.Controllers
     public class OrganisationController : CommonAttributesControllerBase
     {
         private readonly IManageApi _manageApi;
+        private readonly IFeatureFlags featureFlags;
```
Comment (timabell) [inline src/ui/Controllers/OrganisationController.cs]: I think this should be `_featureFlags` by c# conventions.
Also means you don't need `this.` in the constructors.

minor

**Comment by dankmitchell** [comment]
🚢

**Pull request merged**